docs/src/components/header.js

```diff
@@ -1,70 +1,234 @@
+    display: none;
+  }
 `
 
-const Nav = styled.nav``
+const MenuText = styled.span`
+  margin-right: 0.7rem;
+  font-size: 1.7rem;
+`
+
+const CloseLink = styled.a`
+  display: block;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 2.5rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    display: none;
+  }
+`
 
-const Navitems = styled.ul`
+const StyledUnsortedList = styled.ul`
   list-style: none;
   margin: 0;
+  padding: 0;
 `
 
-const Navitem = styled.li`
-  display: inline-block;
-  margin: 0 1rem;
+const StyledListItem = styled.li`
+  display: inline;
+  padding-right: 1.5rem;
+  padding-bottom: 1rem;
+  padding-left: 1.5rem;
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    padding-bottom: 1.5rem;
+    border-bottom: ${props =>
+      props.active ? '0.5rem solid #fff' : '0.5rem solid #45c2f0'};
+
+    &:last-child {
+      margin-left: auto;
+    }
+  }
+`
+
+const StyledExternalLink = styled(External)`
+  text-decoration: none;
+  color: ${props => (props.active ? '#45c2f0' : '#fff')};
+
+  ${util.createMediaQuery(util.defaultBreakpoints[0])} {
+    color: white;
+  }
 `
 
-export default ({ siteTitle }) => (
+const BoxedListItem = ({ children, href, active, ...props }) => (
+  <Box mb={['0.7rem', '0.1rem']} width={[1, 'auto']} {...props}>
+    <StyledListItem active={active}>
+      <StyledExternalLink href={href} active={active}>
+        {children}
+      </StyledExternalLink>
+    </StyledListItem>
+  </Box>
+)
+
+export default () => (
   <PanelProvider>
     <Root>
       <Wrapper>
-        <Heading>
-          <Link to="/">
-            <Logo src={dta} />
-            {false && <Banner>Notify</Banner>}
-          </Link>
-          <Nav>
-            <Navitems>
-              <Navitem>
-                <Link to="/">Support</Link>
-              </Navitem>
-              <Navitem>
-                <Link to="/">Features</Link>
-              </Navitem>
-              <Navitem>
-                <Link to="/">Pricing</Link>
-              </Navitem>
-            </Navitems>
-          </Nav>
-        </Heading>
+        <Flex justifyContent="space-between">
+          <BannerLink to="/">
+            <Flex
+              justifyContent="flex-start"
+              p={['1rem 0', '3rem 0']}
+              alignItems="center"
+            >
+              <Logo src={coa} />
+              <HeaderName>Notify</HeaderName>
+              <Badge>Alpha</Badge>
+            </Flex>
+          </BannerLink>
+          <OpenMenuLink href="#nav" aria-label="Open navigation">
+            <MenuText>Menu</MenuText>
+            <MenuIcon />
+          </OpenMenuLink>
+        </Flex>
+
+        <Nav id="nav">
+          <CloseLink href="#" aria-label="Close navigation">
+            <CloseIcon />
+          </CloseLink>
+          <StyledUnsortedList>
+            <Flex flexWrap="wrap" m={[0, '1rem 0 1rem -1.5rem']}>
+              <Box width={[1, 'auto']}>
```

Comment:
Can you look into the `width` attribute being forwarded on to the DOM? It shouldn't be—I've come across this before but can't recall why it does it. Grid should just use it to determine the CSS but shouldn't be forwarding it to the DOM element.